test/PolledChannelTest.cpp

```diff
@@ -0,0 +1,159 @@
+#include "pipert/PolledChannel.h"
+#include "gtest/gtest.h"
+
+#include "pipert/Scheduler.h"
+
+ class PolledChannelTest : public ::testing::Test {
+  protected:
+    pipert::Scheduler scheduler_;
+
+    PolledChannelTest() : scheduler_(1) {
+    }
+
+    virtual ~PolledChannelTest() {
+    }
+
+    virtual void SetUp() {
+    }
+
+    virtual void TearDown() {
+    }
+
+  };
+
+TEST_F(PolledChannelTest, PolledChannelCreationTestWithScheduler) {
+  pipert::Scheduler scheduler(1);
```

Comment:
Is there a special reason for using a separat scheduler instead of shceduler_?

Reply:
No, the test cases are not ready yet. I just created the PR to help the debug of #9.